front/src/views/routes/workspace/admin/database/DatabaseTable.vue

```diff
@@ -392,30 +415,37 @@ export default {
       })
       return result
     },
-    editableColumns () {
-      if (!this.block.definition.columns) return []
-      return this.block.definition.columns.filter(c => this.isEditableColumn(this.crudMode, c))
-    },
+    /**
+     * Get the current view with table_actions and columns
```

Comment:
I would prefer not use verb to describe the function, as it's a computed property, and not a method.
It returns something, but, this is not a "get" or "retrieve", I think ?

```suggestion
     * Current view with table_actions and columns
```